packages/tools/generator-widget/generators/app/templates/packages/package_web.json.ejs

```diff
@@ -23,8 +23,9 @@
     "lint:fix": "pluggable-widgets-tools lint:fix",<% if (hasUnitTests) { %>
     "test": "pluggable-widgets-tools test:unit:web --no-cache --ci<% if (hasE2eTests) { %> && npm run test:e2e<% } %>",
     "test:unit": "pluggable-widgets-tools test:unit:web --coverage",<% } %><% if (hasE2eTests) { %>
-    "test:e2e": "pluggable-widgets-tools test:e2e --url \"https://mywidget.mxapps.io\"",
-    "test:e2e:dev": "pluggable-widgets-tools test:e2e:web:dev",<% } %>
+    "pretest:e2e": "npm run release && node ../../../scripts/test/updateAtlas.js --latest-atlas",
```

Comment:
This doesn't look like intended change, users don't have to update atlas I think (lucky them) :)